bigtable/client/testing/table_integration_test.cc

```diff
@@ -71,6 +71,16 @@ std::vector<bigtable::Cell> TableIntegrationTest::ReadRows(
   return result;
 }
 
+std::vector<bigtable::Cell> TableIntegrationTest::MoveCellsFromReader(
+    bigtable::RowReader& reader) {
+  std::vector<bigtable::Cell> result;
+  for (auto const& row : reader) {
+    std::copy(row.cells().begin(), row.cells().end(),
```

Comment:
Consider using `std::move()` (the function in `<algorithm>` not the function in `<utility>`)

Reply:
And if you think it is not a good idea or not possible tell me here!

Reply:
done.